### DISCUSSION TITLE
KWin use of CPU

### DISCUSSION THREAD

**Discussion by grahamperrin** [helloSystem/hello#379] [General]
Did someone report excessive CPU usage? 

I worked around this by upgrading the OS and packages: 

![image](https://user-images.githubusercontent.com/192271/189568513-7e28c1ec-22a9-4434-9128-5986c620ab2f.png)

https://github.com/helloSystem/KWin

**Comment by probonopd** [comment]
KWin CPU usage is ~0.5% on my system, so nothing out of the expected.
